config/samples/crdb-tls-example.yaml

```diff
@@ -19,7 +19,7 @@ kind: CrdbCluster
 metadata:
   name: crdb-tls-example
 spec:
-  cockroachDBVersion: v21.1.7
+  cockroachDBVersion: v21.1.8
```

Comment:
IIRC we don't want 21.1.8 to be an option. @udnay can you confirm this was the release that prevents rollbacks?

Reply:
@udnay ping